.github/pull_request_template.md

```diff
@@ -0,0 +1,25 @@
+Fixes #ISSUE.
+
+
+## Pre-merge checklist
+
+<!--- A common case for documentation is the addition of new `make` goals. Descriptions should be documented for new goals both in the (i) `help` command at the bottom of the `mondo-ingest.Makefile` and (ii) `docs/developer/workflows.md`. -->
```

Comment:
```suggestion
<!--- Docs: A common case for documentation is the addition of new `make` goals. Descriptions should be documented for new goals both in the (i) `help` command at the bottom of the `mondo-ingest.Makefile` and (ii) `docs/developer/workflows.md`. -->
```

Just adding the word "Docs" so that it's easy to tell at a glance what this comment applies to. I also suggest removing the blank line in between this comment and docs.

Note that I see we can't put the comment below `- [ ] Docs` because that changes the formatting of the bullets.